src/pyroboplan/planning/prm.py

```diff
@@ -123,20 +133,35 @@ def construct_roadmap(self, sample_generator=None):
             if check_collisions_at_state(self.model, self.collision_model, q_sample):
                 continue
 
+            radius = self.options.max_neighbor_radius
+            k = self.options.max_neighbor_connections
+
+            # If using PRM* dynamically compute the radius and k for this addition
+            if self.options.prm_star:
+                n = len(self.graph.nodes)
+                d = len(q_sample)
+                if n > 0:
+                    radius = radius * (np.log(n) / n) ** (1 / d)
+                    k = int(k * np.log(n))
```

Comment:
Pedantic request to avoid single_letter variables unless it's like a loop index or something?

Or, if this matches up with the PRM* literature, maybe just point to that resource?

Reply:
This I can do.